packages/migration-graph/package.json

```diff
@@ -28,7 +28,7 @@
     "vitest": "^0.23.4"
   },
   "peerDependencies": {
-    "typescript": ">4.0"
```

Comment:
It's better to leave this 4.0
If we pin a higher version it will just be annoying during the running of `upgrade`.

ref: https://github.com/rehearsal-js/rehearsal-js/issues/305

Reply:
This doesn't appear to be an issue. Take a look at the attached video; this is the current rehearsal build from this PR.


https://user-images.githubusercontent.com/10506014/213488292-83a6a376-0ec1-4d64-9452-ad61ef9324f7.mov

Reply:
@lynchbomb after release:

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @rehearsal/migrate@1.0.2-beta
npm WARN Found: peer typescript@">4.9" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer typescript@">4.9" from the root project
npm ERR! code ETARGET
npm ERR! notarget No matching version found for typescript@>4.9.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```